BGPViewerCore/Service/BGPHeService.cs

```diff
@@ -131,26 +131,32 @@ private IEnumerable<string[]> ExtractAsnsInfoFromPeerTable(IWebElement peerTable
             }
         }
 
-        private IEnumerable<string[]> ExtractPrefixAndAsnInfoFromNetInfoDivTable(string tableInnerHtml) 
+        private IEnumerable<string[]> ExtractPrefixAndAsnInfoFromNetInfoDivTable(IWebDriver driver) 
```

Comment:
ExtractPrefixAndAsnInfoFromNetInfoDivTable () could return an object with AsNumber, Prefix and Name properties too. This way it'll be clear what is beeing returned by the code.